### ISSUE TITLE
Stats for message sent to collection

### ISSUE THREAD

**Issue by abhi1203** [glific/glific-frontend#1667]
**Is your feature request related to a problem? Please describe.**
[Requirements](https://docs.google.com/document/d/1Q7ozYAHl1wcVQlXcg3iZjQZPHaUQJzQCYHTLaajncrs/edit#bookmark=id.berdvy700npg)

**Describe the solution you'd like**
[Design](https://www.sketch.com/s/ba323b85-11d7-4687-8a19-681a32132bab/a/Qba1p4x)

**Comment by kurund** [comment]
@abhi1203 

We are showing message actions on click of chip icon, so to be consistent can we move view stats as one of the actions?

**Comment by abhi1203** [comment]
@kurund i realised that the dropdown is not frequently used option. and hence tred to reduce clicks required to reach the stats - also brings the feature clearly in sight - so better discoverability. I would recommend putting it upfront if there are no other challenges.

**Comment by kurund** [comment]
@abhi1203 

I agree it will be more visible.

I was thinking more in terms of UI consistency for all chat interfaces. I think if `view stats` is something useful then users will start using chip icons.

Also, the text `view stats` after each message is a bit odd IMO.

**Comment by abhi1203** [comment]
@kurund this will be available only on chat screen for collections, not for individual chats. Any other suggestions to make it visible and quickly accessible - maybe having an icon above or below the dropdown? or maybe when they hover on the message - we show it somewhere at the right bottom corner of the message.

**Comment by kurund** [comment]
Yes, it won't be available on Individual chats for now, but there is a use case for that too ( Not now )

Can we please have it part of the actions for now? Adding icons will add more confusion.

**Comment by abhi1203** [comment]
@kurund i understand, but if it's not a challenge can we please put it upfront. the thing is going through the cycle of understanding from the user testing is also longer to find out if there's any challenge. I totally understand your point about consistency but im trying to prioritise usability/discoverability for this one. Also trying to course correct 😅 for a few other challenges in the UI we came to know by interacting to users.

**Comment by kurund** [comment]
@abhi1203 

Since you strongly feel this is important let go with it.

**Comment by abhi1203** [comment]
Thanks @kurund

**Comment by mdshamoon** [comment]
@abhi1203 there's a pending state for message apart from success and failure. Can we add that as well in the design

**Comment by abhi1203** [comment]
Okay sure @mdshamoon

**Comment by kurund** [comment]
@mdshamoon 

Can you please discuss this during the sprint with the team and decide on the next steps?

**Comment by mdshamoon** [comment]
Discussed with the team and we will brainstorm a solution for this as this is needed by many organizations.
